metaflow/plugins/kfp/tests/flows/failure_flow.py

```diff
@@ -27,13 +27,6 @@ def start(self):
             output = subprocess.check_output(command, shell=True)
             print(str(output))
         else:
-            command = (
-                f"./kubectl get workflow {os.environ.get('MF_ARGO_WORKFLOW_NAME')} "
-                f"--namespace {os.environ.get('POD_NAMESPACE')} -o yaml"
-            )
-            print(f"{command=}")
-            output = subprocess.check_output(command, shell=True)
-            assert "pod deleted" in str(output)
```

Comment:
Agree that this can be removed. 

Adding more info - when reproducing the error behavior, we found that sometimes the deleted pod doesn't show up in the returned yaml from  `./kubectl get workflow {os.environ.get('MF_ARGO_WORKFLOW_NAME') --namespace {os.environ.get('POD_NAMESPACE')} -o yaml` instead of showing up as `pod deleted`. This may be a bug from argo?